tests/unit/test_network_connections.py

```diff
@@ -3260,6 +3260,15 @@ def test_set_deprecated_slave_type(self):
             self.assertTrue("port_type" in connection)
             self.assertTrue("slave_type" not in connection)
 
+    def test_mac_address_argvalidator(self):
+        """
+        Test that argvalidator for validating mac address is correctly defined.
+        """
+        validator = network_lsr.argument_validator.ArgValidatorMac("mac")
+        self.assertEqual(
+            validator._validate_impl(41135085296, "mac"), "52:54:00:12:34:56"
```

Comment:
This should probably call validator.validate() instead of the internal method `_validate_impl()`.

Reply:
I called `validator.validate()` instead